src/Compilers/Core/Portable/StrongName/StrongNameProvider.cs

```diff
@@ -1,9 +1,71 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+using System.Diagnostics;
 using System.IO;
+using System.Reflection.Metadata;
+using System.Security.Cryptography;
+using Microsoft.Cci;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
+
+
+    internal enum SigningCapability
+    {
+        SignsStream,
+        SignsPeBuilder,
+    }
+
+    internal class IOOperations {
```

Comment:
This name is too easy to confuse with IOperations. I'd prefer IFileSystem 

Also should be in own file.